pombola/south_africa/urls.py

```diff
@@ -273,7 +273,8 @@
     url(
         r'^person/(?P<person_slug>[-\w]+)/messages/$',
         WriteToRepresentativeMessages.as_view(),
+        {'configuration_slug': 'south-africa-assembly'},
         name='sa-person-write-all'
     ),
-    url(r'^write/', include('pombola.writeinpublic.urls')),
+    url(r'^write/', include('pombola.writeinpublic.urls'), {'configuration_slug': 'south-africa-assembly'}),
```

Comment:
Feedback: Change the final parameter to `kwargs=` if that is equivalent to make it clearer what's going on.